core/api/workspace.josh.hcl

```diff
@@ -165,17 +171,28 @@ struct "log-description" {
 }
 
 struct "event" {
-	field "id" "string" {}
-	field "log" "string" {}
-	field "timestamp" "string" {}
-	field "message" "string" {}
+  field "id" "string" {}
+  field "log" "string" {}
+  field "timestamp" "string" {}
+  field "message" "string" {}
 }
 
 struct "process-description" {
-	field "id" "string" {}
+  field "id" "string" {}
   field "provider" "string" {}
-	field "name" "string" {}
+  field "name" "string" {}
+  field "running" "bool" {}
+}
+
+struct "component-status" {
+	field "component-id" "string" {}
 	field "running" "bool" {}
+  field "envVars" "map[string]string" {}
+	field "CPUPercent" "float64" {}
```

Comment:
fields in these josh files should use `kebab-case`, as that is unambiguous when translating to other cases.

You should add "CPU" to the `inflect.go` file, so you get the correct capitalization in codegen for that.